### ISSUE TITLE
TreeSetTimeContext(NULL,NULL,NULL) still crashes within mdsip

### ISSUE THREAD

**Issue by GabrieleManduchi** [MDSplus/mdsplus#1222]
you can reproduce the issue launching mdsip manually and then connecting via TDIC

TDIC>mdsconnect(<host:port>')
TDIC>SetTimeContext(*,*,*)     ---->mdsip crashes

I tried to debug it but but the code is not clear to me and ---sorry to repeat-- macros don't help!!.

**Comment by zack-vii** [comment]
hmm did you install on the server and restart the service
if I do 
mdsconnect('localhost')
mdsvalue('SetTimeContext()')

no problem

**Comment by zack-vii** [comment]
it seems there has not been an alpha released yet with the fix included the newest version in git does have it fixed though. but you cannot get it via the repo yet.

**Comment by GabrieleManduchi** [comment]
Can you explain the last sentence?????

**Comment by zack-vii** [comment]
sure. in order to get the fix via e.g.
```bash
apt-get update
apt-get -y upgrade
```
the alpha build needs to successfully build and release the latest version.. but it failed. last night
so the patch was not available yet.
you can build the current alpha though to have/test the fix already

**Comment by tfredian** [comment]
The fix for calling TreeSetTimeContext when no tree is currently open was merged into alpha but the alpha build last night had a failure in one of the alpine build tests which prevented a new release from being issued with the TreeSetTimeContext fix. Timo is working on preventing the intermittent failure of alpine builds and once that is merged we will be building a new alpha release.

**Comment by GabrieleManduchi** [comment]
Can you let me know when the fix will be available on alpha? We have some urgency in having it working.
